text/warn.md

```diff
@@ -0,0 +1,9 @@
+You are receiving this message because you have broken some rules or are on the verge of doing so.
```

Comment:
I'd rather have a warning for each violation than this, so it's clear to the user and anyone seeing this command called. For example, `?warn @user for inappropriate-image` will be clear to everyone, and the target user gets a more detailed warning in DM. We shouldn't try to come up with this upfront, and expand as needed.

Reply:
Okay, how about this:

- `?warn @mention --reason=promote`: DM warning about promoting external products/services
- `?warn @mention --reason=spoonfeed`: DM warning about repeated attempts to get people to spoonfeed the user (and draining everyone's energy in the process)
- `?warn @mention --reason=inappropriate`: DM warning about inappropriate behavior (nudity, violence, etc.)

Reply:
I was thinking of using options only for optional parameters. In this example, something like `?warn @user inappropriate-image --severity=minor`.

Reply:
Noted with thanks. I'll go with `?warn @mention {promote,spoonfeed,inappropriate}` for now.